### ISSUE TITLE
Handling "not found in Search documents"

### ISSUE THREAD

**Issue by brittnylapierre** [crkn-rcdr/Access-Platform#600]
https://access.canadiana.ca/object/edit/69429/s06q1sf2n91s  

This is that odd situation where a noid is a member of a collection, but the noid doesn't exist any more. Something worth thinking about what you think the tool should do in that case -- show the staff member at the top ic "manage members" in some way?  

Automatically clear it out silently?

**Comment by brittnylapierre** [comment]
Hey @RussellMcOrmond automatically clear it out silently is the preferred option (automation.) Let me know what the todos are for that at some point, and we can also coordinate where this should go in the backlog! Thanks

**Comment by RussellMcOrmond** [comment]
It is also possible this is bad data from an old version, and this won't happen in the future.  If it isn't easy to do in the front-end tool, then I can write a back-end tool that fixes the data and then we can forget about it.

**Comment by RussellMcOrmond** [comment]
I incorrectly characterised this issue.  It was staring me in the fact, and it was for code I wrote...  :blush:

Guess putting my mind so far into the past when I'm trying to think about the future is hard. :confused:


> Noids not found in **Search** documents

What Hammer is doing is checking if noids referenced in a collection are available in `cosearch` documents, as a way to avoid "Not found" errors when a patron clicks on a link.

I looked some of these up, and they exist: but aren't yet published.


A better solutions is to complete a noid lookup tool https://github.com/crkn-rcdr/Access-Platform/issues/524 , and then train staff to look up noids they are warned about and ensure they are published (or remove them from the collection -- they will know the slug from the noid lookup).
